CONTRIBUTING.md

```
@@ -4,31 +4,221 @@ Thank you for your interest in contributing to StellarStream! This guide will he
+
+2. **Install all dependencies** (backend + frontend)
+   ```bash
+   npm run install:all
+   ```
 
+3. **Set up environment variables**
+   ```bash
+   cp backend/.env.example backend/.env
+   ```
+   Edit `backend/.env` and fill in required values. For local-only development (no deployed contract), set:
+   ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)